Kafka not starting with JDK 14

I'm still trying to get CI on JDK 14:
- [x] REST Assured
- [x] Gradle
- [ ] Now Kafka is not starting properly apparently:

```
Caused by: java.lang.RuntimeException: Unable to start Quarkus test resource io.quarkus.it.kafka.KafkaTestResource@c11332b
	at io.quarkus.test.common.TestResourceManager.start(TestResourceManager.java:51)
	... 44 more
Caused by: java.lang.RuntimeException: kafka.zookeeper.ZooKeeperClientTimeoutException: Timed out waiting for connection while in state: CONNECTING
	at io.quarkus.it.kafka.KafkaTestResource.start(KafkaTestResource.java:30)
	at io.quarkus.test.common.TestResourceManager.start(TestResourceManager.java:46)
	... 44 more
Caused by: kafka.zookeeper.ZooKeeperClientTimeoutException: Timed out waiting for connection while in state: CONNECTING
	at kafka.zookeeper.ZooKeeperClient.$anonfun$waitUntilConnected$3(ZooKeeperClient.scala:259)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at kafka.utils.CoreUtils$.inLock(CoreUtils.scala:253)
	at kafka.zookeeper.ZooKeeperClient.waitUntilConnected(ZooKeeperClient.scala:255)
	at kafka.zookeeper.ZooKeeperClient.<init>(ZooKeeperClient.scala:113)
	at kafka.zk.KafkaZkClient$.apply(KafkaZkClient.scala:1858)
	at kafka.server.KafkaServer.createZkClient$1(KafkaServer.scala:375)
	at kafka.server.KafkaServer.initZkClient(KafkaServer.scala:399)
	at kafka.server.KafkaServer.startup(KafkaServer.scala:207)
	at io.debezium.kafka.KafkaServer.startup(KafkaServer.java:220)
	at java.base/java.util.concurrent.ConcurrentHashMap$ValuesView.forEach(ConcurrentHashMap.java:4780)
	at io.debezium.kafka.KafkaCluster.startup(KafkaCluster.java:250)
	at io.quarkus.it.kafka.KafkaTestResource.start(KafkaTestResource.java:28)
	... 45 more
```

--- a/bom/runtime/pom.xml
+++ b/bom/runtime/pom.xml
@@ -131,7 +131,7 @@
         <mutiny-client.version>0.0.13</mutiny-client.version>
         <kafka2.version>2.4.1</kafka2.version>
         <debezium.version>1.1.0.Final</debezium.version>
-        <zookeeper.version>3.4.14</zookeeper.version>
+        <zookeeper.version>3.5.7</zookeeper.version>
         <!-- Scala is used by Kafka so we need to choose a compatible version -->
         <scala.version>2.12.9</scala.version>
         <tika.version>1.22</tika.version>

